lib/whatsapp_sdk/resource/interactive_action.rb

```diff
@@ -0,0 +1,41 @@
+# typed: strict
+# frozen_string_literal: true
+
+module WhatsappSdk
+  module Resource
+    class InteractiveAction
+      extend T::Sig
+
+      # Returns the buttons of the Action. For reply_button type, it's required.
+      #
+      # @returns buttons [Array<InteractiveActionButton>] .
+      sig do
+       returns(T::Array[InteractiveActionButton])
+      end
+      attr_accessor :buttons
+
+      # TODO: attr_accessor :button
+      # TODO: attr_accessor :sections
+      # TODO: attr_accessor :catalog_id
+      # TODO: attr_accessor :product_retailer_id
```

Comment:
Are you planning to add them in a follow up PR?

Reply:
These should definitely be done in a follow up PR as the scope of this PR is to add support for reply buttons only.

I see myself working of the usecases of catalog and products soon. When I do, I can lend a helping hand then to complete this module.

Reply:
Thank you